recipes-kernel/kernel-tests/kernel-test-nohz.bb

```diff
@@ -40,7 +40,7 @@ pkg_postinst_ontarget:${PN}-ptest:append() {
     ISOLATED_CPU=$((CPUS - 1))
 
     if [ $ISOLATED_CPU -gt 0 ]; then
-        echo 'set otherbootargs="${otherbootargs} isolcpus='$ISOLATED_CPU' nohz_full='$ISOLATED_CPU' mitigations=off intel_pstate=disable intel_idle.max_cstate=0 processor.max_cstate=0 nosoftlockup mce=ignore_ce audit=0 tsc=nowatchdog"' > /boot/runmode/no-hz-full-params.cfg
+        echo 'set otherbootargs="${otherbootargs} isolcpus=nohz,domain,managed_irq,'$ISOLATED_CPU' nohz_full='$ISOLATED_CPU' mitigations=off intel_pstate=disable intel_idle.max_cstate=0 processor.max_cstate=0 nosoftlockup mce=ignore_ce audit=0 tsc=nowatchdog"' > /boot/runmode/no-hz-full-params.cfg
```

Comment:
Not entirely relevant to this change, but it appears that `isolcpus` has been deprecated since ni/linux@b0d40d2b22fe.

Reply:
Yeah, it has been marked deprecated and at some point it may get removed, however:

- It is still present in the latest Linus master (6.7-rc1 at this point)
- It has users that oppose it's removal: https://lore.kernel.org/linux-rt-users/CANV1gkdPJ=gqT2nUuGzN=ytOxHN92opoS1aGEDqy3FmTWTmusw@mail.gmail.com/
- We will notice when it gets removed (via this test)
- I would rather keep this test simple for now and not have to add support for the intermediate cgroup support state we're currently in until LV RT upgrades to v2